lib/backend/k8s/conversion.go

```diff
@@ -68,13 +68,13 @@ func (c converter) parsePolicyNameNamespace(name string) (string, error) {
 
 // parsePolicyNameNetworkPolicy extracts the Kubernetes Namespace and NetworkPolicy that backs the given Policy.
 func (c converter) parsePolicyNameNetworkPolicy(name string) (string, string, error) {
-	// Policies backed by NetworkPolicies have form "np.projectcalico.org/<ns_name>.<np_name>"
-	if !strings.HasPrefix(name, "np.projectcalico.org/") {
+	// Policies backed by NetworkPolicies have form "knp.default.<ns_name>.<np_name>"
+	if !strings.HasPrefix(name, "knp.default.") {
 		// This is not backed by a Kubernetes NetworkPolicy.
 		return "", "", fmt.Errorf("Policy %s not backed by a NetworkPolicy", name)
 	}
 
-	splits := strings.SplitN(strings.TrimPrefix(name, "np.projectcalico.org/"), ".", 2)
+	splits := strings.SplitN(strings.TrimPrefix(name, "knp.default."), ".", 2)
```

Comment:
Hm, this logic is actually already wrong.  If there's a namespace or a NetworkPolicy with a `.` in it, this will break.

Some examples:
- namespace="casey.davenport" -> `casey.davenport.mypolicy`
- policy="my.policy" -> `default.my.policy`

We should add UTs for these and then fix up the code, but let's do it in a separate PR immediately after this one since it's not a regression caused by this PR.